src/backend/marsha/core/serializers/video.py

```diff
@@ -396,40 +406,56 @@ def validate(self, attrs):
                                 "authentication."
                             }
                         )
+                # We can identify the user for this context_id and consumer_site, we make sure
+                # this user hasn't already registered for this video.
+                if LiveRegistration.objects.filter(
+                    consumer_site=attrs["consumer_site"],
+                    deleted=None,
```

Comment:
This is not needed. The `safedelete` package manage this for us. Or if it's the case we have lot of cases in the application where it's missing. Can you confirm it with a test ?

Reply:
true, no need to specify deleted